packages/ra-ui-materialui/src/button/SortButton.tsx

```diff
@@ -71,8 +71,8 @@ const SortButton: FC<{ fields: string[]; label?: string }> = ({
     };
 
     const buttonLabel = translate(label, {
-        field: translate(`resources.products.fields.${currentSort.field}`),
-        order: translate(`ra.sort.${currentSort.order}`),
+        field: currentSort.field,
```

Comment:
no, we must grab the `resource` from the listcontext, and replace `products` with it

Reply:
Right